tests/pkg_lora-serialization/tests-lora_serialization.c

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
I'm not sure how nitpicky this is, but usually our c-files containing the `main` functions are called `main.c`

Reply:
Same here, I would prefer a main.c.